### ISSUE TITLE
Is it a bug or a feature that default tensorboard shows a scalar 'epoch'?

### ISSUE THREAD

**Issue by lity** [PyTorchLightning/pytorch-lightning#1662]
pytorch-lightning version:0.7.5

Before I upgraded from 0.7.3 to 0.7.5, there's no 
such a scalar 'epoch'  on tensorboard.

**Comment by mateuszpieniak** [comment]
I will also join the discussion with a similar issue i.e. the default TensorBoard shows an empty `loss` graph in TensorBoard when I decided not to log `loss` per batch level in `training_step`. 

![image](https://user-images.githubusercontent.com/31375424/80621031-03d7b200-8a47-11ea-8f90-ac654ddcf0ef.png)

**Comment by olineumann** [comment]
Could you please post you code. I just tested tensorboard with version 0.7.5. No issue here.

![image](https://user-images.githubusercontent.com/12863612/80627532-61bcc780-8a50-11ea-9887-7b612f9729ac.png)

**Comment by mateuszpieniak** [comment]
@olineumann Sorry for trouble :facepalm: It seems like if I have one experiment with `{'log': loss}` in `training_step` and the other one without it, then tensorboard executed inside `lightning_logs` make a union of all keys to display. Therefore for one experiment is blank as in the attached image.

**Comment by lity** [comment]
> Could you please post you code. I just tested tensorboard with version 0.7.5. No issue here.
> 
> ![image](https://user-images.githubusercontent.com/12863612/80627532-61bcc780-8a50-11ea-9887-7b612f9729ac.png)

my code:
`    def training_step(self, batch, batch_nb):
        x, y = batch
        y_hat = self.forward(x)
        loss = self.criterion(y_hat, y)
        tensorboard_logs = {'train_loss': loss}
        return {'loss': loss, 'log': tensorboard_logs}`

`    early_stop_callback = pl.callbacks.EarlyStopping(
        monitor='val_loss',
        min_delta=0.00,
        patience=5,
        verbose=False,
        mode='min'
    )
    checkpoint_callback = pl.callbacks.ModelCheckpoint(save_top_k=-1)
    trainer = pl.Trainer(
        # train_percent_check=0.0001,
        # val_percent_check=0.001,
        num_sanity_val_steps=0,
        max_epochs=40,
        progress_bar_refresh_rate=10,
        benchmark=True,
        gpus=1,
        precision=16,
        early_stop_callback=early_stop_callback,
        checkpoint_callback=checkpoint_callback,
    )
    trainer.fit(waterdrop_model)`

The 'epoch' tag on tensorboard is still there after "validation_step, validation_epoch_end, test_step, test_epoch_end" were commented.

my tensorboard:
![screenshot-mechrevo_6006-2020 04 30-08_03_07](https://user-images.githubusercontent.com/3319045/80659875-c7469d80-8abc-11ea-9eac-eae3b3fe9f5c.png)

**Comment by lity** [comment]
> feature :)
> 
> it’s super useful to always track epoch

Thanks , I see.  Please take a look at my another issue: https://github.com/PyTorchLightning/pytorch-lightning/issues/1663
